modules/microshift-install-rpms-low-latency.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-low-latency.adoc
```

Comment:
Is this right?
```suggestion
// * microshift_configuring/microshift-low-latency.adoc
```

Reply:
Actually, it should be
// microshift_configuring/microshift_low_latency/microshift-low-latency.adoc
since I made a directory for it now